templates/careers/application/index.html

```diff
@@ -82,8 +82,8 @@ <h5 class="p-notification__title">We could not withdraw your application.</h5>
       <div class="row">
         <div class="col-12">
           <h1 class="p-heading--2 u-sv2"><strong>{{ application["candidate"]["first_name"] }} {{ application["candidate"]["last_name"] }}</strong><br>Application for {{ application["job_post"]["title"] or application["jobs"][0]["name"] }}</h1>
-          {% if "stage_progress" in application and application["stage_progress"]["assessment"] and application['source']['id'] == 2 %}
-            <p class="u-no-margin--bottom"><a href="/careers/{{ application['job_post_id'] }}" class="p-link--inverted">Job description for this role</a></p>
+          {% if "stage_progress" in application and application["stage_progress"]["assessment"] and application['source']['id'] == 2 and application['job_post_id'] %}
+            <p class="u-no-margin--bottom"><a href="/careers/{{ application['job_post_id'] }}" class="p-link--inverted">Read more about this job&nbsp;&rsaquo;</a></p>
```

Comment:
This is super minor, but are we using "role" or "job" throughout the site? I think all of the careers pages use the former